frontend/packages/dev-console/src/components/pipelines/pipeline-overview/TriggerLastRunButton.tsx

```diff
@@ -20,14 +20,16 @@ const TriggerLastRunButton: React.FC<TriggerLastRunButtonProps> = ({
   const latestRun = usePipelineRunWithUserLabel(
     getLatestRun({ data: pipelineRuns }, 'startTimestamp'),
   );
-  const { label, callback, accessReview } = rerunPipelineAndStay(PipelineModel, latestRun);
+  const { label, callback, accessReview } = rerunPipelineAndStay(PipelineRunModel, latestRun);
   const isAllowed = useAccessReview(accessReview, impersonate);
   return (
-    isAllowed && (
-      <Button variant="secondary" onClick={callback} isDisabled={pipelineRuns.length === 0}>
-        {label}
-      </Button>
-    )
+    <Button
+      variant="secondary"
+      onClick={callback}
+      isDisabled={!isAllowed || pipelineRuns.length === 0}
```

Comment:
This will show the button for View-only users. This breaks the first criteria of the sidebar.